src/main.ts

```diff
@@ -5,24 +5,28 @@ import Bunyan2Loggly from "./loggly";
 
 // configure bunyan logging module for reaction server
 // See: https://github.com/trentm/node-bunyan#levels
-const levels = ["FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE"];
+const levels = ["fatal", "error", "warn", "info", "debug", "trace"];
 
-// set stdout log level
-let level = process.env.REACTION_LOG_LEVEL || "INFO";
+const customLevel = process.env.REACTION_LOG_LEVEL;
+
+let level: Bunyan.LogLevel = "info";
+if (customLevel && levels.includes(customLevel)) {
+  level = customLevel as Bunyan.LogLevel;
+}
 
 // allow overriding the stdout log formatting
-// available options: short|long|simple|json|bunyan
 // https://www.npmjs.com/package/bunyan-format
-const outputMode = process.env.REACTION_LOG_FORMAT || "short";
-
-level = level.toUpperCase();
+const outputModes = ["short", "long", "simple", "json", "bunyan"];
+type OutputMode = "short" | "long" | "simple" | "json" | "bunyan";
```

Comment:
I'm doing this because Bunyan type definitions do not export the Options interface and if the outputMode here does not match the string literal type defined in the BunyanFormat options, the compiler gives a type check error when instantiating the default stream on line 29.